packages/svelte/src/internal/client/dom/elements/attributes.js

```diff
@@ -341,9 +341,10 @@ export function set_dynamic_element_attributes(node, prev, next, css_hash) {
  * because updating them through the property setter doesn't work reliably.
  * In the example of `width`/`height`, the problem is that the setter only
  * accepts numeric values, but the attribute can also be set to a string like `50%`.
- * If this list becomes too big, rethink this approach.
+ * If this list becomes too big, rethink this approach. In case of draggable trying to
+ * set `element.draggable='false'` will actually set draggable to `true`
```

Comment:
Would be better to first name the draggable case, then the "if this list becomes too big" sentence - that way things are grouped better

Reply:
Fixed it thanks